scipy/optimize/tests/test_cython_optimize.py

```diff
@@ -0,0 +1,93 @@
+"""
+test cython optimize zeros api functions
+"""
+
+from __future__ import division, print_function, absolute_import
+import numpy as np
+from scipy.optimize.cython_optimize import _zeros
+
+
+EXPECTED_BISECT = [
+    1.259765625,
+    1.279296875,
+    1.298828125,
+    1.318359375,
+    1.337890625,
+    1.357421875,
+    1.376953125,
+    1.392578125,
+    1.408203125,
+    1.427734375]
+
+
+# test bisect
+def test_bisect():
+    assert np.allclose(EXPECTED_BISECT,
+                       list(_zeros.loop_example('bisect')))
+
+
+EXPECTED_RIDDER = [
```

Comment:
Why is `EXPECTED_RIDDER` different from `EXPECTED_BISECT`?  Is the function the same or different for  each of the tests?  If the same, then the answer shouldn't depend on the solver, right?